components/text/aspell/patches/01-gcc7.patch

```diff
@@ -0,0 +1,22 @@
+--- aspell-0.60.6.1/modules/filter/tex.cpp.orig	Tue May 16 14:08:00 2017
++++ aspell-0.60.6.1/modules/filter/tex.cpp	Tue May 16 14:08:28 2017
+@@ -174,7 +174,7 @@
+ 
+     if (c == '{') {
+ 
+-      if (top.in_what == Parm || top.in_what == Opt || top.do_check == '\0')
++      if (top.in_what == Parm || top.in_what == Opt || *top.do_check == '\0')
+ 	push_command(Parm);
+ 
+       top.in_what = Parm;
+--- aspell-0.60.6.1/prog/check_funs.cpp.orig	Tue May 16 14:06:49 2017
++++ aspell-0.60.6.1/prog/check_funs.cpp	Tue May 16 14:07:30 2017
+@@ -647,7 +647,7 @@
+     }
+   }
+   if (i == width-1) {
+-    if (word == '\0')
++    if (word[0] == '\0')
```

Comment:
Do we really check for non-NULL or current symbol != '\0' here?

Reply:
I kind of expected that the person would not use \0 as a null pointer but now I see that it might have been the case.